x-pack/plugins/maps/public/classes/sources/vector_source/vector_source.tsx

```diff
@@ -34,7 +34,7 @@ export interface GeoJsonWithMeta {
   meta?: GeoJsonFetchMeta;
 }
 
-export interface BoundsFilters {
+export interface LayerDataFilters {
```

Comment:
Can you explain this name change? Why LayerDataFilters? What does that mean?

Reply:
good point. I found the union-type in makeSearchSource confusing. Thought a generic-name could work better, because the object is just a subset of the syncContext.

To align, what about rename to `BoundsRequestMeta`: ?
```async makeSearchSource(
    searchFilters: VectorSourceRequestMeta | VectorJoinSourceRequestMeta | BoundsRequestMeta,
```

Reply:
++ to BoundsRequestMeta